spec/datadog/profiling/exporter_spec.rb

```diff
@@ -24,6 +25,8 @@
     collector
   end
   let(:internal_metadata) { { no_signals_workaround_enabled: no_signals_workaround_enabled } }
+  let(:info) { { profiler: { running_under_test: true } } }
+  let(:info_collector) { instance_double(Datadog::Profiling::Collectors::Info, info: info) }
   let(:no_signals_workaround_enabled) { false }
   let(:logger) { Datadog.logger }
   let(:options) { {} }
```

Comment:
Minor: May be worth changing the `'returns a flush containing the data from the recorders'` to also validate it includes the info (and I guess the internal_metadata too, which I realized I missed :P)